### PR TITLE
Patches from Debian

### PR DESCRIPTION
**Pull request by charles2910**
Hi, as we discussed in #2, I've finished to work on the build process (it builds like a charm now - except it complains about using old configure.in instead of configure.ac and other minor things you can see [here in the top of the log](https://salsa.debian.org/charles/devtodo/-/jobs/950817)). So now I'm forwarding the patches applied to the source code in Debian.

All the commit messages are from the description of the patches, I've added a couple words in one or two of them and there are links to the bugs submitted in Debian bug tracker that the patchs fix.

### CODE REVIEW

**Comment by alecthomas** [comment]
Thanks so much for this. Just one question then I'm happy to merge.

File: configure.in [hunk 1]
```diff
@@ -1,6 +1,10 @@
-AC_INIT(src/main.cc)
+AC_INIT
+AC_CONFIG_SRCDIR([src/main.cc])
 AM_INIT_AUTOMAKE(devtodo,0.1.20)
 
+# don't rebuild Makefiles (=> timestamp skew due to patch)
+AM_MAINTAINER_MODE
```
[the remark this answers is not in the document]
Comment (charles2910) [reply configure.in:6]: Yes, you're right. It should be just for packaging purposes. I will remove that change.

[the remark this answers is not in the document]
Comment (charles2910) [reply configure.in]: It's fixed in 7fe549b

**Comment by alecthomas** [comment]
Thanks!

**Pull request merged**